packages/@aws-cdk/aws-codepipeline/test/integ.pipeline-cfn.expected.json

```diff
@@ -221,7 +215,13 @@
             ],
             "Name": "CFN"
           }
-        ]
+        ],
+        "ArtifactStore": {
+          "Location": {
+            "Ref": "PipelineArtifactsBucket22248F97"
+          },
+          "Type": "S3"
+        }
```

Comment:
Can you revert this change? Ordering should not matter for integ tests.

Reply:
Done